.github/workflows/ci.yml

```diff
@@ -192,7 +193,10 @@ jobs:
           name: nuget-packages
           path: ./Bin/Release/ILGPU*.${{ steps.package.outputs.version }}.*nupkg
 
-  publish:
+    outputs:
+      tag: ${{ steps.version.outputs.tag }}
```

Comment:
As per the comment above, we should probably just use `version` here instead of the new variable `tag`.